src/Groups/GAPGroups.jl

```diff
@@ -1590,6 +1592,83 @@ function _map_word_syllable(vi::Pair{Int, Int}, genimgs::Vector, genimgs_inv::Ve
 end
 
 
+@doc Markdown.doc"""
+    syllables(g::FPGroupElem)
+
+Return the syllables of `g` as a list of pairs `(gen,exp)` where
```

Comment:
```suggestion
Return the syllables of `g` as a list of pairs `gen => exp` where
```
Maybe use the pair syntax?